### PR TITLE
Fix DREF change request - Issue #1948

### PR DESCRIPTION
**Pull request by puranban**

### CODE REVIEW

**Review by frozenhelium — APPROVED** [review]

File: src/root/components/OperationalUpdatePdfDocument/index.tsx [hunk 1]
```diff
@@ -485,17 +485,70 @@ function OperationalUpdatePdfDocument(props: Props) {
...
+          operationalUpdateResponse?.event_scope ||
+          operationalUpdateResponse?.anticipatory_actions ||
+          operationalUpdateResponse?.images_details.length > 0
+        ) && (
+            <View>
+              <Text style={pdfStyles.sectionHeading}>
+                {strings.drefExportDescriptionOfTheEvent}
+              </Text>
+              {operationalUpdateResponse.images_details?.map((el) => (
+                <View style={pdfStyles.subSection}>
```
Comment (tnagorra) [inline src/root/components/OperationalUpdatePdfDocument/index.tsx:500]: key is missing

File: src/root/components/OperationalUpdatePdfDocument/index.tsx [hunk 2]
```diff
@@ -485,17 +485,70 @@ function OperationalUpdatePdfDocument(props: Props) {
...
+                </View>
+              )}
+              {operationalUpdateResponse?.event_description && (
+                <View style={pdfStyles.subSection}>
+                  <Text style={pdfStyles.subSectionHeading}>
+                    {isImminentOnset ?
+                      strings.drefExportImminentWhereWhenHow
+                      :
+                      strings.drefExportWhatWhereWhen}
+                  </Text>
```
Comment (tnagorra) [inline src/root/components/OperationalUpdatePdfDocument/index.tsx:524]: This is a weird formatting choice

File: src/root/components/DrefPdfDocument/index.tsx [hunk 3]
```diff
@@ -470,51 +473,71 @@ function DrefPdfDocument(props: Props) {
...
+                  />
+                </View>
+              ))}
+              {dref?.event_description && (
+                <View style={pdfStyles.subSection}>
+                  <Text style={pdfStyles.subSectionHeading}>
+                    {isImminentOnset ?
+                      strings.drefExportImminentWhereWhenHow
+                      :
+                      strings.drefExportWhatWhereWhen}
```
Comment (tnagorra) [inline src/root/components/DrefPdfDocument/index.tsx:503]: Weird formatting choice

File: src/root/components/DrefPdfDocument/index.tsx [hunk 4]
```diff
@@ -470,51 +473,71 @@ function DrefPdfDocument(props: Props) {
...
+                </View>
+              )}
+            </View>
+          )}
         <View style={pdfStyles.poSection}>
           <Text style={pdfStyles.sectionHeading}>
             {strings.drefExportPreviousOperations}
           </Text>
+          <View>
+            <Link src={resolveUrl(window.location.origin, 'https://go.ifrc.org/preparedness#operational-learning')}>
```
Comment (tnagorra) [inline src/root/components/DrefPdfDocument/index.tsx:537]: Any reason to pass complete url to resolveUrl?

File: src/root/components/DrefPdfDocument/index.tsx [hunk 5]
```diff
@@ -997,7 +1037,7 @@ function DrefPdfDocument(props: Props) {
           </Link>
         </View>
       </PDFPage>
-    </Document>
+    </Document >
```
Comment (tnagorra) [inline src/root/components/DrefPdfDocument/index.tsx:1040]: We should not add a whitespace here

File: src/root/components/OperationalUpdatePdfDocument/index.tsx [hunk 6]
```diff
@@ -569,6 +622,22 @@ function OperationalUpdatePdfDocument(props: Props) {
...
+              {
+                display: 'flex',
+                flexDirection: 'row',
+                justifyContent: 'center'
+              }
+            ]}
+            >
+              {operationalUpdateResponse?.photos_details?.map((el) => (
+                <PDFImage
+                  style={pdfStyles.mapImage}
```
[the remark this answers is not in the document]
Comment (puranban) [reply src/root/components/OperationalUpdatePdfDocument/index.tsx:636]: fixed

File: src/root/components/OperationalUpdatePdfDocument/index.tsx (hunk 1 above)
Comment (puranban) [reply src/root/components/OperationalUpdatePdfDocument/index.tsx:500]: fixed

File: src/root/components/OperationalUpdatePdfDocument/index.tsx (hunk 2 above)
Comment (puranban) [reply src/root/components/OperationalUpdatePdfDocument/index.tsx:524]: can you please suggest the formatting type for this case?

**Pull request merged**